mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -1,20 +1,33 @@
 use std::collections::HashMap;
 
 use futures_util::future::{self, join_all};
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+use tokio::sync::mpsc::{self, UnboundedSender};
 use tracing::{error, info};
 
 use crate::bridge::{Bridge, BridgeError};
 use crate::settings::Settings;
 
 /// Controller that handles the settings and monitors changes, spawns new Bridges and monitors shutdown signal.
-#[derive(Default)]
 pub struct BridgeController {
     bridges: HashMap<String, Bridge>,
```

Comment:
I think there is going to be an issue with this new structure where we don't store the hashmap of bridges. This was eliminated because we were going to use Rc types within the bridge internally, but switched back to locking because of some errors spawning threads. 

If we think we are going to keep the locks, we should have a PR to add back the hashmap of bridges, or you can add it back here.

Reply:
removed the hashmap as it's not needed